docs/resources/file.md

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "Storage"
+---
+# databricks_file Resource
+
+This resource allows uploading and downloading files in [databricks_volume](volume.md) up to 5GiB in octet-stream.
```

Comment:
I would put `up to 5GiB in octet-stream` as a Note as the size may change over time.

Reply:
Also, add a note that UC Volumes is only current implementation and that the list of supported destinations may change